curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-use-modal-verbs/67d6b95f099bbe66d89b11af.md

```diff
@@ -0,0 +1,96 @@
+---
+id: 67d6b95f099bbe66d89b11af
+title: Task 98
+challengeType: 22
+dashedName: task-98
+---
+
+<!-- REVIEW -->
+
+# --description--
+
+This is a review of the entire dialogue you just studied.
+
+# --instructions--
+
+Place the following phrases in the correct spot:
+
+`critical`, `performance`, `overwhelmed`, `might`, `have`, `deployment`, and `depends on`.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`Sophie: Hi, James. We're discussing some of the possible outcomes for our next software release. I have a few questions about what BLANK happen if we run into issues. Can you help me understand the possibilities?`
+
+`James: Sure, Sophie. What do you want to know?`
+
+`Sophie: Well, I'm concerned about the BLANK process. It could go smoothly, or we might face some bugs. What do you think is most likely?`
+
+`James: It's hard to say. We could have a smooth deployment, but there's always a chance that a few bugs might pop up. It BLANK how well we've tested everything.`
+
+`Sophie: That makes sense. If we find bugs during deployment, what would be the immediate response?`
+
+`James: If we find BLANK bugs, we'll probably BLANK to roll back the deployment and fix them before trying again. It might delay the release, but it's better than having a broken product.`
```

Comment:
```suggestion
`James: If we find BLANK bugs, we'll probably BLANK roll back the deployment and fix them before trying again. It might delay the release, but it's better than having a broken product.`
```